src/lsp/protocol.ts

```diff
@@ -258,4 +258,15 @@ export namespace GetRemoteProjectNames {
   );
 }
 
+export interface TokenGenerationParams {
+  serverPath?: string;
+  errorMessage?: string;
+}
+
+export namespace GetTokenGenerationParams {
+  export const type = new lsp.RequestType<string, TokenGenerationParams, null>(
```

Comment:
Minor - why `TokenGenerationParams`? Isn't the request for `GetTokenGenerationUrl`

Reply:
Yup good catch, this needs to be updated to the latest signature from the LS 👍🏻